dev/com.ibm.ws.jpa_ol_fat/test-framework/src/com/ibm/ws/testtooling/msgcli/DataPacket.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.testtooling.msgcli;
```

Comment:
Any reason you use io.openliberty for package prefix in the other source directories, but in this one you use com.ibm.ws?

Reply:
Because the test tooling is a "common set" of tools between all my fats (wish we could have a library project that fats could use, but I understand that is discouraged?) which I copied from one of the other projects.